src/main/proto/wfa/measurement/internal/kingdom/measurement_log_entry.proto

```diff
@@ -43,44 +43,51 @@ message MeasurementLogEntry {
 
     // Time that the error occurred.
     google.protobuf.Timestamp error_time = 2;
+  }
 
+  message Details {
     // Human-readable error message. This should not include any sensitive info.
```

Comment:
Maybe a nit, but is every log_message an error message?

Reply:
No, that's a good catch. I copied this comment from elsewhere and forgot to adjust.